lib/LedgerSMB/Report/Dates.pm

```diff
@@ -216,7 +216,7 @@ sub _get_to_date {
     if ($self->interval eq 'month'){
        $date->add(months => 1);
     } elsif ($self->interval eq 'quarter'){
-       $date->add(months => 3);
+       $date->add(months => 3);         ## no critic ( ProhibitMagicNumbers)
```

Comment:
Sorry about that.  I meant to mention it in the comment above.  At one point, I wanted something to push to Github this was it. 

Do you think that in context this is not very magical and so no-critic works?  MONTHS_PER_QUARTER is more correct.  Which is preferred,  Readonly or constant or
something else?